core/src/Authorization.html

```diff
@@ -169,17 +169,18 @@
             ...TOP_NAV_DEFAULTS.logout,
             ...logoutItem
           };
-          isProfileLogoutItem = !!logoutItem;
+
+          isProfileLogoutItem = Boolean(logoutItem);
           profileLogoutFnDefined = false;
           AuthLayerSvc.setProfileLogoutFn(null);
-          if (logoutItem) {
-            const profileLogoutfn = logoutItem.customLogoutFn;
+          if (
+            logoutItem &&
+            GenericHelpers.isFunction(logoutItem.customLogoutFn)
+          ) {
             // TODO: PROFNAVLOGOUT: three smiliar implementations.
             // TODO: whats the difference between this profileLogoutFn, profileNav, auth-layer
-            if (profileLogoutfn && GenericHelpers.isFunction(profileLogoutfn)) {
-              profileLogoutFnDefined = true;
-              AuthLayerSvc.setProfileLogoutFn(profileLogoutfn);
-            }
+            profileLogoutFnDefined = true;
+            AuthLayerSvc.setProfileLogoutFn(profileLogoutfn);
```

Comment:
I think `setProfileLogoutFn` should get `logoutItem.customLogoutFn` as parameter otherwise profileLogoutfn is not defined. See removed line 176